sources/ingest/importers/utils/shared.py

```diff
@@ -7,6 +7,14 @@ class LanguageString:
     fi: str = None
     sv: str = None
     en: str = None
+    text: str = None
+    defaultLanguage: str = "FI"
+
+    def __init__(self, **kwargs):
+        self.fi = kwargs["fi"]
+        self.sv = kwargs["sv"]
+        self.en = kwargs["en"]
+        self.text = kwargs[self.defaultLanguage.lower()]
```

Comment:
Isn't this implementation prone to the same error case that it fixes? It becomes a lot less likely, but with an input `fi = None, sv = None, en = None`, doesn't `text = None`.

The schema that the graphql server validates against is here: https://github.com/City-of-Helsinki/unified-search/blame/76453faad0a594a11a475a13553e597c50328669/graphql/src/schemas/shared.ts#L16-L23

The `text` field is a required string. I guess `None` maps to `null`. An empty string could pass the validation.

My other question is that does it make sense to retain these fields? I remember there being some doubt about their use.

I guess that the purpose of the `text` field is to provide a value for use in contexts where the locale is unclear. But based on this implementation it seems like ensuring it has a value may be difficult. Placing an empty string in it to pass validation sounds counterintuitive.

Reply:
Okay, so instead of trying to implement the missing text and default language fields, they should be totally taken away? Fine by me, because I couldn't find any use for them.

Reply:
Earlier, the text was never even tried to be filled...

Reply:
Yup, was thinking the same. No one should be getting any benefit out of using these fields because most likely they are empty.

If there's no reason to have the fields, then it'd make sense to prune them so that we don't maintain code that doesn't have a usecase.

Reply:
There is now an alternative PR for this #78